app/views/wikies/index.html.erb

```diff
@@ -17,7 +17,7 @@
       <tr>
         <td><%= wiky.title %></td>
         <td><%= wiky.body %></td>
-        <td><%= wiky.user %></td>
+        <td><%= User.find(wiky.user_id).email %></td>
```

Comment:
Try to avoid executing db queries within a view. DB queries are better off elsewhere, often in the model or in the controller. Can you think of an alternative way of getting the email from `wiky.user` do no query is necessary?